test/runtests.jl

```diff
@@ -189,7 +196,7 @@ import LinearAlgebra.SingularException
             CliffordAlgebra(1,1,1)
         ) 
 
-        cl = CliffordAlgebra(1,1,1)
+        # cl = CliffordAlgebra(1,1,1)
```

Comment:
Before this PR all tests ran for `CliffordAlgebra(1,1,1)` instead of looping over multiple different algebras.